x-pack/test/detection_engine_api_integration/security_and_spaces/group1/get_prebuilt_rules_status.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from 'expect';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import {
+  createRule,
+  deleteAllRules,
+  deleteAllTimelines,
+  getPrebuiltRulesStatus,
+  getSimpleRule,
+  installPrebuiltRulesFromMocks,
+} from '../../utils';
+import {
+  createPrebuiltRuleAssetSavedObjects,
+  ELASTIC_SECURITY_RULE_ID,
+  RulesPackageType,
+  SAMPLE_PREBUILT_RULES,
+  UNIQUE_HISTORICAL_RULES_COUNT,
+} from '../../utils/create_prebuilt_rule_saved_objects';
+import { deleteAllPrebuiltRules } from '../../utils/delete_all_prebuilt_rules';
+import { incrementPrebuiltRuleVersion } from '../../utils/increment_prebuilt_rule_version';
+
+const PACKAGE_TYPES: RulesPackageType[] = ['historical', 'latest_only'];
+const RULES_COUNT_BY_PACKAGE_TYPE: Record<RulesPackageType, number> = {
+  latest_only: SAMPLE_PREBUILT_RULES.length,
+  historical: UNIQUE_HISTORICAL_RULES_COUNT,
+};
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const log = getService('log');
+
+  describe('get_prebuilt_rules_status', () => {
+    beforeEach(async () => {
+      await deleteAllPrebuiltRules(es);
+      await deleteAllRules(supertest, log);
+      await deleteAllTimelines(es);
+    });
+
+    it('should return empty structure when no rules package installed', async () => {
+      const body = await getPrebuiltRulesStatus(supertest);
+
+      expect(body).toMatchObject({
+        rules_custom_installed: 0,
+        rules_installed: 0,
+        rules_not_installed: 0,
+        rules_not_updated: 0,
+        timelines_installed: 0,
+        timelines_not_installed: expect.any(Number), // Timelines are shipped on the filesystem, so they are always available to install
+        timelines_not_updated: 0,
+      });
+    });
+
+    it('should show that one custom rule is installed when a custom rule is added', async () => {
+      await createRule(supertest, log, getSimpleRule());
+
+      const body = await getPrebuiltRulesStatus(supertest);
+      expect(body).toMatchObject({
+        rules_custom_installed: 1,
+        rules_installed: 0,
+        rules_not_installed: 0,
+        rules_not_updated: 0,
+        timelines_installed: 0,
+        timelines_not_installed: expect.any(Number),
+        timelines_not_updated: 0,
+      });
+    });
+
+    PACKAGE_TYPES.forEach((packageType) => {
+      describe(`package type: ${packageType}`, () => {
+        it('should display the number of rules and templates available to install', async () => {
```

Comment:
```suggestion
        it('should return the number of rules available to install', async () => {
```

It looks like we don't really test the number of templates. Worth doing that?